.asf.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+github:
+  description: "Mirror of Apache Qpid ProtonJ2"
+  homepage: https://qpid.apache.org
```

Comment:
protonj2 does not seem to have components page on the website

Reply:
https://qpid.apache.org/proton/index.html

Reply:
I somehow expected to find protonj2 on the same level as qpid jms at https://qpid.apache.org/components/